src/docs/product/projects/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Projects
+sidebar_order: 1
+description: "View all the projects associated with teams that you're a member of, then dive into their details quickly."
+---
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+A _Project_ represents a service or application. For example, you might have separate projects for your API server and frontend client.
+
+You can create a project for a service and select the particular language or framework when you first start using Sentry. You can add more projects on an as-needed basis. For example, if you want to manage access permissions and/or control privacy and data settings. Setting up multiple projects to reflect your application landscape with finer granularity will go a long way in helping with visibility and triage.
+
+When displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, you are looking at information that is immediately relevant to your work.
+
+Projects differ from [environments](/product/sentry-basics/environments/), which are designed to help triage issues, especially in a multi-staged release process.
+
+## Project Homepage
+
+The project homepage lists the projects by teams of which you're a member, providing a high-level overview:
+
+- A snapshot of both errors and transactions
+- The team and team members associated with each project (remember, projects can be associated to multiple teams, so a project may be listed more than once)
+- Deploys, if you have configured your SDK to [provide a release identifier](/platform-redirect/?next=/configuration/releases/) and are tracking [deploys](/product/cli/releases/#creating-deploys)
+
+The projects are displayed in alphabetical order, separated by team. **Note:** For faster access, star your favorites to ensure they display at the top of the page each time you view.
+
+## View Project Details
+
+Project details combine errors, performance, and health in one central place. The [Project Details](/product/projects/project-details/) page provides a critical overview to developers, product owners, and engineering managers to answer the central question of "What is the project's current state and where do I need to look next?".
```

Comment:
Next try:

```suggestion
The [Project Details](/product/projects/project-details/) page provides an overview over errors, performance, and the health of your releases. It allows to quickly correlate spikes in error rates with the issues causing them react to critical alerts, and analyze long-term trends.
```

Reply:
Suggest the last sentence be "You can use the page to quickly correlate spikes in error rates with the issues causing them, react to critical alerts, and analyze long-term trends.

Reply:
Added, thanks!